### DISCUSSION TITLE
Change the search bar

### DISCUSSION THREAD

**Discussion by shipurjan** [storybookjs/storybook#29010] [Ideas]
### Is your feature request related to a problem? Please describe.

I'm always frustrated when I have to use the search bar in a storybook with a lot of components.

When I search for a component in a storybook with many components it scrolls down and I have to scroll up to re-search every time if I had selected wrong one. 

For example I search for 'Table' and I get many components like 'TableRow', 'TableCell' etc. 

When I click on one then the results from the search bar are reset and return to normal, unfiltered view. And to re-search for the same query I have to scroll up and reactivate the search bar.

### Describe the solution you'd like

1. Make search bar sticky
2. Don't reset the search results when a result is clicked

### Are you able to assist to bring the feature to reality?

no

**Comment by shaedrich** [comment]
Yeah, this is a huge PITA every time 👍🏻
